internal/registry/clientsecretrequest/rest.go

```diff
@@ -70,32 +125,159 @@ func (*REST) Categories() []string {
 }
 
 func (r *REST) Create(ctx context.Context, obj runtime.Object, createValidation rest.ValidateObjectFunc, options *metav1.CreateOptions) (runtime.Object, error) {
```

Comment:
@benjaminapetersen @cfryanr it might be a good idea to pass in the delegated authorizer of this aggregated API server and check if the calling user can `get` + `update` the associated OIDC client.  RBAC does not allow the easy expression of "you can create but only with this name", and having the subject access review check would allow the admin to create an OIDC client and give read and write access to that single client to a developer.  Then if the admin also gave "create client secret request" access to that developer, they would be able to manage that single OIDC client but not others.